lua/pointshop/providers/tmysql.lua

```diff
@@ -1,120 +1,113 @@
 --[[
-	Pointshop tmysql4 Adapter by xbeastguyx
-		- Meant to follow a similar pattern to the original
-	
-	Setup: 
-	
-		Install the tmysql4 module:
-			https://facepunch.com/showthread.php?t=1442438
-			Instructions for installation are provided
+Pointshop tmysql4 Adapter by xbeastguyx
+- Meant to follow a similar pattern to the original
 
-		Set PS.Config.DataProvider = "tmysql"
-]]--
+Setup:
 
+Install the tmysql4 module:
+https://facepunch.com/showthread.php?t=1442438
+Instructions for installation are provided
 
+Set PS.Config.DataProvider = "tmysql"
+]]
+--
 local MySQL_Host = ""
 local MySQL_User = ""
 local MySQL_Pass = ""
 local MySQL_Database = ""
 local MySQL_Port = 3306
-
-
 -- Do not edit below
-
 local db, err
```

Comment:
why even remove whitespace like this and everything else? It forces reviewers to use `?w=1` or some other method to see the real changes in the patch.